website/docs/r/autoscaling_policy.html.markdown

```diff
@@ -184,6 +238,38 @@ The following arguments are supported:
 * `predefined_metric_type` - (Required) Describes a scaling metric for a predictive scaling policy. Valid values are `ASGAverageCPUUtilization`, `ASGAverageNetworkIn`, `ASGAverageNetworkOut`, or `ALBRequestCountPerTarget`.
 * `resource_label` - (Required) A label that uniquely identifies a specific Application Load Balancer target group from which to determine the request count served by your Auto Scaling group.
 
+##### customized_scaling_metric_specification
+The following arguments are supported:
+
+* `metric_data_queries` - (Required) A list of up to 10 structures that defines custom scaling metric in predictive scaling policy
+
+##### customized_load_metric_specification
+The following arguments are supported:
+* `metric_data_queries` - (Required) A list of up to 10 structures that defines custom load metric in predictive scaling policy
+
+##### customized_capacity_metric_specification
+The following arguments are supported:
+* `metric_data_queries` - (Required) A list of up to 10 structures that defines custom capacity metric in predictive scaling policy
+
+##### metric_data_queries
+The following arguments are supported:
+* `id` - (Required) A short name for the metric used in predictive scaling policy.
+* `expression` - (Optional) The math expression used on the returned metric. You must specify either `expression` or `metric_stat`, but not both.
+* `metric_stat` - (Optional) A structure that defines CloudWatch metric to be used in predictive scaling policy. You must specify either `expression` or `metric_stat`, but not both.
+* `label` - (Optional) A human-readable label for this metric or expression.
+* `return_data` - (Optional) A boolean that indicates whether to return the timestamps and raw data values of this metric, the default it true
+
+##### metric_stat
+The following arguments are supported:
+* `metric` - (Required) A structure that defines the CloudWatch metric to return, including the metric name, namespace, and dimensions.
+* `stat` - (Required) The statistic of the metrics to return.
+* `unit` - (Optional) The unit of the metrics to return.
+
+##### metric
+* `metric_name` - (Required) The name of the metric.
+* `namespace` - (Required) The namespace of the metric.
+* `dimensions` - (Optional) The dimensions of the metric.
```

Comment:
Can we add an extra sentence to prompt readers to refer to the `metric_dimension` section since this argument has nested arguments

Reply:
Yeah, I added a section of `dimensions` definition as I realized it was using a different name. Even though the content is the same.